### DISCUSSION TITLE
Great options:

### DISCUSSION THREAD

**Discussion by RazvanGavriliu** [any-block/any-block#29] [Q&A]
Great processors - **list2table** and **list2tab** !!!
But I'm not not able to render the images ... it strange that list2tab is able to render videos ...

![image](https://github.com/LincZero/obsidian-any-block/assets/24705879/85e2c785-d1a2-4e6e-85cb-45a2a7e2fc82)

![image](https://github.com/LincZero/obsidian-any-block/assets/24705879/59b07415-77bc-4604-9803-d102b1eeb66b)

The same is available for `[scroll]` - it is rendering the video, but not the images.

**Is there anything I'm not doing right so that images to be rendered with list2tab processor?**

Thanks!

**Comment by LincZero** [comment]
I know the bug. This is a bug in `HTMLElement ReaderMarkdown("md text")` in the Obsidian API interface. If `md text` contains an image, the image in the returned HTML element will not be rendered properly.

If the official does not fix this interface bug, you need to manually process the image link yourself, which will be more difficult to fix.

**Comment by LincZero** [comment]
This is a bug in the Obsidian API itself, and in November '23 they used a new API to replace the previous buggy version. The latest version of AnyBlock has also fixed this bug
